qtpy/tests/test_qtserialbus.py

```diff
@@ -0,0 +1,28 @@
+"""Test QtSerialBus."""
+
+import pytest
+from qtpy import (
+    parse,
+    PYSIDE6,
+    PYSIDE_VERSION,
+    QtBindingMissingModuleError,
+)
+
+
+@pytest.mark.skipif(not PYSIDE6 or parse(PYSIDE_VERSION) >= parse('6.5'), reason='.')
+def test_qtserialbus_pyside6_below_6_5():
+    with pytest.raises(QtBindingMissingModuleError) as excinfo:
```

Comment:
```suggestion
    with pytest.raises(QtBindingMissingModuleError):
```

No need for the `as` if you don't actually use `excinfo`, no? Or maybe you're planning to?

Reply:
I *was* considering inspecting the actual error in the test. Do you have an opinion on whether that's worth doing?

Reply:
Perhaps, yeah, at least the custom attributes of the error object (as opposed to the message, which is considered a user-facing implementation detail subject to change and a test of which would be too fragile).